### ISSUE TITLE
Issues with Inverse kinematics and Pinned center

### ISSUE THREAD

**Issue by psychoanima** [opentoonz/opentoonz#2368]
### Issue Summary
Pinned center with IK bones is not locking bones properly, although lock option is activated

### Video Reference
https://1drv.ms/v/s!AtMv1SrHrjT38m2lnCUmt6kDz7ZO

### Steps to Reproduce
Lock at least two bones at different frames 

### Expected Results
Pinned bone should stay locked while other bones will be able to move

### Actual Results
Pinned bone is not locked (reverse effect)

### System Information
* **OpenToonz Version**:  1.2.1 (built jun 28) 
* **Operating System**: Windows 10 Pro
* **CPU**: i7 2630QM
* **Memory**: 20 gb
* **Graphics Card**:  NVidia Quadro 1000m

**Comment by beeheemooth** [comment]
I have about the same problem with two legs.

https://groups.google.com/forum/#!category-topic/opentoonz_en/troubleshooting/Yf5sL_mbgQ8

![ezgif-4-cd6d8dca54](https://user-images.githubusercontent.com/25724133/48094927-14291b00-e21c-11e8-83e5-edcc108cff07.gif)

**Comment by DarrenTAnims** [comment]
I've not been able to use IK because of this. It's never worked for me. I meant to log a bug, but never found time and I just used other methods to animate with.

**Comment by DarrenTAnims** [comment]
I've not been able to use IK because of this. It's never worked for me. I meant to log a bug, but never found time and I just used other methods to animate with. So thanks for logging these, with examples.

**Comment by psychoanima** [comment]
I hope someone will assign to this issue and fix it so we can use IK normally :)

**Comment by turtletooth** [comment]
@psychoanima @beeheemooth Thanks for posting examples here.  I would personally love for IK to be usable, but I don't touch it because the results are so unreliable.

**Comment by artisteacher** [comment]
I agree - using IK is very unpredictable. Adding constraints to limit the range of motion would help in some areas - though probably not with pinning.
